### PR TITLE
Pick up incomplete bridged proposals too

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: scenario/utils/bridgeProposal.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  const searchBlocks = timelockBuf;
+  const block = await deploymentManager.hre.ethers.provider.getBlockNumber();
+  const filter = receiver.filters.ProposalCreated();
+  const logs = await fetchLogs(receiver, filter, block - searchBlocks, block);
+  const proposals = [];
+  if (logs) {
+    for (let log of logs) {
+      const [, id, , , , , eta] = log.args;
+      const state = await receiver.state(id);
+      if ([BridgedProposalState.Queued,].includes(state)) {
```
Comment (scott-silver) [inline scenario/utils/bridgeProposal.ts:20]: nit: extra comma `[BridgedProposalState.Queued,]`

**Review by hayesgm — APPROVED** [review]
cool

File: scenario/utils/index.ts [hunk 2]
```diff
@@ -352,12 +352,13 @@ export async function executeOpenProposal(
...
   }
 
-  const proposal = await governor.proposals(id);
-  await setNextBlockTimestamp(dm, proposal.eta.toNumber() + 1);
-
-  // Execute proposal (w/ gas limit so we see if exec reverts, not a gas estimation error)
-  await setNextBaseFeeToZero(dm);
-  await governor.execute(id, { gasPrice: 0, gasLimit: 12000000 });
+  // Execute proposal (maybe, w/ gas limit so we see if exec reverts, not a gas estimation error)
+  if (state == ProposalState.Queued) {
```
Comment (kevincheng96) [inline scenario/utils/index.ts:356]: This is probably the breaking change. You'll want to set `state` as `Queued` after queuing it in the block above if we add this check.

**Review by kevincheng96 — APPROVED** [review]
LGTM!

**Pull request merged**